crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -611,10 +611,11 @@ def cancel
       end
 
       def upgrade_pacemaker_cluster(cluster_env)
-        founder = ::Node.find(
-          "pacemaker_founder:true AND " \
+        founder_name = ::Node.find(
+          "pacemaker_founder:* AND " \
```

Comment:
This clause is not necessary, as you're searching for cluster nodes only.

Reply:
+1